frontend/src/metabase/admin/datamodel/components/FilterPopover/FilterPopover.unit.spec.tsx

```diff
@@ -77,7 +77,7 @@ describe("FilterPopover", () => {
       it("should have an operator selector", () => {
         setup({ filter: NUMERIC_FILTER });
         expect(screen.getByText("Equal to")).toBeInTheDocument();
-        expect(screen.getByText("1,234")).toBeInTheDocument();
+        expect(screen.getByText("1234")).toBeInTheDocument();
```

Comment:
We decided not to render the value using internal components in [this Slack thread](https://metaboat.slack.com/archives/C0645JP1W81/p1717599783880869). Fixing this is possible, but requires us reimplementing some default components from Mantine as they are not exported.